docs/devguide/translation.rst

```diff
@@ -0,0 +1,43 @@
+.. _dev-translate:
+
+============
+Translations
+============
+
+The Python community is a diverse and cosmopolitan place. Our documentation
+should be available for users in many different languages and for many different
+locales.
+
+This document explains how to go about translating this documentation.
+
+Read The Docs
+-------------
+
+Currently we host our documentation on http://readthedocs.org. We use Sphinx to
+make the documentation happen.
+
+The relevant documentation for localisation in ReadTheDocs is here:
+
+https://docs.readthedocs.io/en/latest/localization.html#project-with-multiple-translations
+
+Put simply, each new translation is a separate project, presumably a Sphinx
+based repository on GitHub, that contains the translated resources and is linked
+to ReadTheDocs in the usual way - making sure its locale is set  appropriately.
+
+It's possible for this "core" project to link to these other projects as
+"translations of" this one.
+
+Read the Docs automatically figures out the relationships and displays the
+appropriate links between all the related translation projects.
+
+I want to Translate!
+--------------------
+
+Great stuff! Thank you (in advance) for your support!
+
+* Make a new Sphinx project that mirrors the structure of the ``docs`` directory in this project. Call the project something obvious like: ``microbit-micropython-es`` (for the Spanish translation).
```

Comment:
Unless we decide to use a completely different service it sounds like this is the best way to go with read the docs.
Personally, don't like it all that much, is barely practical if you are well versed in git, but very difficult to explain properly to newer developers (or non-developers!) that might see documentation as a low barrier entry.
I also see it being a bit of a problem for sporadic translation updates, which I will assume is going to be the majority of the cases. People, or a team of people, will make a large time investment to do the initial translation and once the main bulk of the work is done the contributions will dry out. After a while, somebody decides to update a translation, they will `git pull` and if there is a large number of conflicts, so I can imagine 3 things could happen:
1) The contributor donates a much larger amount of time than expected, but gets the job done; we are all happy and we thank him profusely.
2) There is a lot of work, this specific contributor cannot do all of it but need to clear the conflicts, so either each un-translated conflict goes back to English (a bit annoying if it’s a small or non-significant change) or the original translation stays without the updated content. Either way, this specific line of documentation is marked as “resolved” and we lose “git tracking” until a new change comes to the same line.
3) The amount of work is overwhelming, or it feels like an “all or nothing” type of situation and the contributor decides not to do it.